hnn_core/parallel_backends.py

```diff
@@ -371,6 +376,7 @@ def simulate(self, net, n_trials, postproc=True):
         """
 
         # just use the joblib backend for a single core
+        # XXX this does not work: if n_procs==1, empty dipole list returned
```

Comment:
something feels fishy here. I'd rather fix it here than in the example if it's easy enough to do?

Reply:
When you say "I'd" do you really mean "you'd"? ;) I'm a bit stumped about that behaviour. I don't know enough about the context manager to understand how you can nest a `JoblibBackend().simulate` inside a `with MPIBackend...`?

Reply:
okay let me give this a shot after lunch

Reply:
see db0555cb867dc72c 

what would you do without my reviews? ;-)

Reply:
oops I forgot to remove the XXX comment